test/whitelist/binutils/glibc.log

```diff
@@ -0,0 +1,6 @@
+#
+# XXX: Unknwon.
+#
+FAIL: indirect5c dynsym
+FAIL: indirect5d dynsym
+FAIL: Build pr22263-1
```

Comment:
Similarly, needs a new relaxation and maybe a new relocation to eliminate tprel relocations in pie mode.  Also similarly, is an optimization, not a correctness issue.